test-network-function/generic/suite.go

```diff
@@ -160,6 +161,10 @@ var _ = ginkgo.Describe(testsKey, func() {
 		log.Info(testOrchestrator)
 		log.Info(containersUnderTest)
 
+		for _, containersUnderTest := range containersUnderTest {
+			testNodeSelector(context, containersUnderTest.oc.GetPodName(), containersUnderTest.oc.GetPodNamespace())
```

Comment:
Out of interest, what's the reason for calling `oc` via a local shell context, rather than using the existing oc context for the container under test?

Reply:
I don't use the oc shell.
The "interactive.Oc" struct stores the name and namespace of the container I'm testing.

Reply:
Ignore me, an `oc` context doesn't make sense here.